### DISCUSSION TITLE
Error when using cuda

### DISCUSSION THREAD

**Discussion by trancoc** [emgucv/emgucv#591] [Q&A]
I'm using .netcore and installing the latest nuget EMGU, with normal CPU instructions, but when I try to use GPU to process it, I get an error: **OpenCV: The library is compiled without CUDA support**

Anyone with experience in using EMGU library with GPU please help me. Thank you!

**Comment by emgucv** [comment]
If you use nuget packages from nuget.org, you are probably using the CPU only release (Emgu.CV.runtime.windows). You need to use the CUDA compatible release instead (Emgu.CV.runtime.windows.cuda).

Instructions on installing the CUDA compatible nuget packages can be found here:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Open_Source_Release

**Reply by trancoc** [reply]
Thanks for support, But i used Emgu.CV.runtime.windows.cuda from nuget and GPU not yet apply
![Capture](https://user-images.githubusercontent.com/15209651/135952525-3cf7d407-d60e-4d25-9194-3c040daf2fe4.PNG)

**Comment by emgucv** [comment]
You have two runtimes installed. Remove all nuget packages (both 4.5.3 and 4.4.0), then re-install the Emgu.CV.runtime.windows.cuda. Other Emgu.CV nuget packages should be 4.4.0 only.
